docs/monthly/2023-08-monthly.md

```diff
@@ -12,7 +12,7 @@ developments to gather their feedback on proposed plans.
 
 ## Roadmap
 
-This month, the team released version 0.12.0 and the project
+This month, the team released  v.0.12.0, and the project
```

Comment:
We do tag our versions just as `0.12.0` and there is no `v` in our versions. So I would like to keep it just `0.12.0` to match the release/tag name.